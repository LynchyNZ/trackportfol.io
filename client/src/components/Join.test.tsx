import React from 'react';
import { shallow } from 'enzyme';
import JoinComponent from './Join';

describe('Join', () => {
  it('renders the Join component', () => {
    expect(JoinComponent).toMatchSnapshot();
  });

  it('displays the Join title (Create your account)', () => {
    const result = shallow(<JoinComponent />).contains('Create your account');
    expect(result).toBeTruthy();
  });
});
