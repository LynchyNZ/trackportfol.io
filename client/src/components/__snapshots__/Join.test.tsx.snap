// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Join renders the Join component 1`] = `[Function]`;
